app.py

```diff
@@ -14,16 +15,16 @@
 from tinydecred.wallet import Wallet
 from tinydecred.ui import screens, ui, qutilities as Q
 
-# The directory of the tinydecred package.
+# the directory of the tinydecred package
 PACKAGEDIR = os.path.dirname(os.path.realpath(__file__))
 
-# Some commonly used ui constants.
+# some commonly used ui constants
 TINY = ui.TINY
 SMALL = ui.SMALL
 MEDIUM = ui.MEDIUM
 LARGE = ui.LARGE
 
-# A filename for the wallet.
+# a filename for the wallet
```

Comment:
Do you want to make all comments sentences? These are phrases so I removed the capitalization and punctuation. It's not important, I know. Probably trauma from grade school that makes comment on this.

Reply:
I think you're right. If it is a sentence, it should have capitalization and punctuation, but small fragments without either are fine.